### PR TITLE
GPS testing and UML diagram 

### PR DESCRIPTION
**Pull request by ishaan2307**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # redback-orion
-IoT and Data science code
```
Comment (MarsImmortal) [inline README.md:2]: this is the orignal read me file so dont edit this.

**Review by MarsImmortal — CHANGES_REQUESTED** [review]
Have a meeting with me regarding creating a pull req. Need to address some changes.

**Comment by MarsImmortal** [comment]
Ishaan just delete the branch you have created and pull the repo on your device again so its up to date. Further make another folder for IOT and add the sensor name over there in order to make it a bit systematic (in main branch). Text me on teams if you are facing any issues.

**Review by cybermehak — APPROVED** [review]
Already reviewed!

**Pull request closed without merging**